connector_woocommerce/unit/export_synchronizer.py

```diff
@@ -0,0 +1,419 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Tech-Receptives Solutions Pvt. Ltd.
+#    Copyright (C) 2009-TODAY Tech-Receptives(<http://www.techreceptives.com>).
```

Comment:
This code is mostly copied from the connector-magento project (which is fine), could you keep the original copyright?